### PR TITLE
Update Pisth screenshots and tags

### PR DESCRIPTION
**Pull request by ColdGrub1384**
Hi! I updated screenshots and tags for Pisth app.
The app is not coded in Objective-C :).

1. [x] Project URL: https://pisth.github.io
2. [x] Update contents.json instead of README 
3. [x] One project per pull request
4. [x] Avoid iOS or open-source in description as it is assumed
5. [ ] Use this commit title format if applicable: Add app-name by @github-username
6. [x] Use approved format for your entry

### CODE REVIEW

File: contents.json [hunk 1]
```diff
@@ -9453,16 +9453,17 @@
...
       ],
       "description": "SSH and SFTP client",
       "source": "https://github.com/ColdGrub1384/Pisth",
       "screenshots": [
-        "https://pisth.github.io/screenshots/1.PNG",
-        "https://pisth.github.io/screenshots/2.PNG",
-        "https://pisth.github.io/screenshots/3.jpeg",
-        "https://pisth.github.io/screenshots/4.PNG",
-        "https://pisth.github.io/screenshots/5.PNG"
+        "https://pisth.github.io/meta/Screenshots/iPhone%20-%205.8%22/1.png,
```
Comment (dkhamsing) [inline contents.json]: missing a quote before the comma at the end

Comment (ColdGrub1384) [reply contents.json]: Ok, thanks, I will fix it

**Comment by ColdGrub1384** [comment]
Oh, sorry I don't know why I commited as READMEbot, I will fix it

**Comment by ColdGrub1384** [comment]
Ok, all is fixed 😅. I was trying scripts inside this repo for trying to get a preview of changes, I think it was because of that I comitted the second time as READMEbot

**Comment by dkhamsing** [comment]
Thank you 😊 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Pull request merged**